### PR TITLE
Feat : add speak synthesizer using Web Speech API

### PR DESCRIPTION
**Pull request by raisoturu**
Memutar audio untuk kode radio ketika di-klik menggunakan [Speech API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Speech_API).

Already tested using:
- Google chrome 8x.xx, Firefox 81.xx (Manjaro Linux & windows 10)
- Google chrome & Firefox (Android 9 MIUI10, Android 8 AOSP)
- Safari (iOS 13.7) 
- and it plays audio

### CODE REVIEW

**Comment by raisoturu** [comment]
Forget to mention #6

File: pages/radio.js [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
 const url = BASE_PATH + '/radio/';
 
 function LambangPramukaPage() {
+
+  const [canSpeak, setSpeak] = useState(false);
+
+  useEffect(() => {
+    // check if browser support Speech API
+    if('speechSynthesis' in window){
+      setSpeak(true)
```
Comment (mazipan) [inline pages/radio.js:21]: can we use this as a reusable hooks @raisoturu ?
so we can use it on other page easier.

Comment (mazipan) [reply pages/radio.js:21]: I expect we just need to use like this:

```js
const { canSpeak } = useSpeachAPI();
```

Comment (raisoturu) [reply pages/radio.js:21]: ah sure...
where I can put my hooks? 
Should I create a new directory "/utils/hooks.js"?

Comment (mazipan) [reply pages/radio.js:21]: just put on dir `/hooks`, no need to add `utils`

Comment (raisoturu) [reply pages/radio.js:21]: okay

Comment (raisoturu) [reply pages/radio.js]: Can you check my last commit? @mazipan

**Review by mazipan — APPROVED** [review]
Looks good mas

**Comment by raisoturu** [comment]
Oke thx mas.
Btw, this is a cool project and I'm planning to add more "Materi" wkw, so imma gonna flood you with notification hahaha.

**Comment by mazipan** [comment]
I am very welcome, just make sure you create an issue first.
so we can understand the context before you create a PR

**Pull request merged**